.gitignore

```diff
@@ -1,4 +1,44 @@
-/.vscode/
+# OSX trash
```

Comment:
We should either (1) remove this section or (2) include a complete list of macos-related files. 
https://github.com/github/gitignore/blob/master/Global/macOS.gitignore

I would prefer option 1 without any macos users chiming in. @travis / @leseb